src/hsmm_mvpy/utils.py

```diff
@@ -665,182 +665,575 @@ def transform_data(data, participants_variable="participant", apply_standard=Tru
     return data
     
 
-def LOOCV(data, participant, n_events, initial_fit, sfreq, event_width=50):
+def LOOCV(data, init, participant, initial_fit, cpus=None, verbose=False):
```

Comment:
we should probably lower-case this function for consistency, I shouldn't have created it in uppercase...

Reply:
then it needs a different name, I will fix it.